public-docs/intro-glance.md

```diff
@@ -0,0 +1,80 @@
+
+## Installation
+
+Check out our [Installation docs](https://docs.reactioncommerce.com/reaction-docs/master/installation).
+
+## Mobile Device Support
+
+The Reaction web apps use mobile-first design principles, which means they look great in a mobile web browser.
+
+While we do have plans to work on a native mobile app on the [Roadmap](https://reactioncommerce.com/roadmap), we do not have a release date. However, you may create your own native mobile app using the Reaction GraphQL API. If you do so using React Native, you can use any of our Reaction Design System components to get your UI built more quickly.
+
+## Localization and Internationalization
+
+We have [internationalization support](i18n.md) for dozens of languages, including right-to-left formats! We also support currency conversion and localized currency formats.
+
+## Search Engine Optimization
+
+Since 2014 [Google has indexed JavaScript when crawling websites](https://webmasters.googleblog.com/2014/05/understanding-web-pages-better.html). Reaction, however, offers page pre-rendering and product detail page metatag generation out of the box to ensure products are indexed well for web crawlers and search engines:
+
+1. [Prerender.io](https://prerender.io/): Reaction includes integration with Prerender.io out of the box. Prerender.io is a commercial service that will generate static renderings of the application for search engines. All you have to do is provide a key to your site and Prerender will handle the pre-rendering.
+2. [Meteor Chrome Headless Spiderable](https://github.com/artlimes/meteor-chrome-headless-spiderable) with the [Reaction Commerce Caching Plugin](https://github.com/artlimes/reaction-commerce-caching-plugin): This updated Meteor package uses Google's Headless Chrome to crawl pages. Combined with the Reaction-specific caching plugin, the Spiderable package will allow Reaction pages to be crawled by search engines.
+3. [dom](https://github.com/reactioncommerce/reaction/blob/master/imports/plugins/core/dom/client/dom.js): Automatically adds `<meta>` tags are for products using the [dom](https://github.com/reactioncommerce/reaction/blob/release-1.8.0/imports/plugins/core/dom/client/dom.js) core package since 1.8, which uses the title, description and `details` fields of the product to render SEO-friendly data. You may need to use a tool like [SEO Inspector](https://chrome.google.com/webstore/detail/seo-inspector/iejckekdjogeeilmllnabmgkbbmedeal?hl=en) to see this data. Read more on [SEO and Metadata](seo-metadata.md).
+4. The storefront UI starter kit app uses NextJS framework with server side rendering enabled.
+
+Read more about [SEO and Reaction Commerce](https://blog.reactioncommerce.com/how-our-javascript-platform-handles-seo/) in our latest blog post.
```

Comment:
Maybe we want to point at the new SEO checklist gated ebook?

Reply:
Where is that?

Reply:
http://marketing.reactioncommerce.com/acton/media/37362/seo-checklist

Reply:
Also, Vishal has a blog post on SEO that is coming out on Thursday that we might want to point to.

Reply:
Added a link